ingestion/tests/integration/orm_profiler/test_orm_profiler_e2e.py

```diff
@@ -107,8 +107,11 @@ def setUpClass(cls) -> None:
         """
         Prepare Ingredients
         """
-        User.__table__.create(bind=cls.engine)
-        NewUser.__table__.create(bind=cls.engine)
+        try:
```

Comment:
what are you trying to catch here. If it is exception when the table already exists can you be more explicit?